### DISCUSSION TITLE
CTK message box doesn't appear after packaging

### DISCUSSION THREAD

**Discussion by AndrewSafwat1** [Akascape/CTkMessagebox#50] [Q&A] [answered]
Hello, 
I tried packaging a CTK application using pyInstaller and followed steps in [https://customtkinter.tomschimansky.com/documentation/packaging](url), and main application works fine but CTK message box when it is supposed to appear in the middle of the window, it only appear at taskbar and cannot show it.    

![Screenshot 2024-01-24 225952](https://github.com/Akascape/CTkMessagebox/assets/98947690/60328e11-2657-4206-a8f3-53d8f5d78e12)

Here is the command written on cmd:
`pyinstaller --noconfirm --onedir --windowed --add-data "c:\python\python37\lib\site-packages/customtkinter;customtkinter/" -F -w -i "path\to\icon.ico" "path\to\app.py"`

**Comment by Akascape** [comment, marked as answer]
@Akascape 
Pack the CTkMessagebox folder (`add-data`) just like customtkinter folder.

If you want a masterclass of pyinstaller, see this: https://www.akascape.com/coding/best-pyinstaller-guide
